src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -163,8 +176,84 @@ def decision_environments_table(
 def event_streams_table(
     since: datetime, full_path: str, until: datetime, **kwargs
 ):
-    query = _get_query(models.EventStream.objects, since, until)
-    return _copy_table("event_streams", query, full_path)
+    event_streams = (
+        models.EventStream.objects.filter(
+            Q(created_at__gt=since, created_at__lte=until)
+            | Q(modified_at__gt=since, modified_at__lte=until)
+        )
+        .order_by("id")
+        .distinct()
+    ).values(
+        "id",
+        "organization_id",
+        "name",
+        "event_stream_type",
+        "eda_credential_id",
+        "additional_data_headers",
+        "test_mode",
+        "test_content_type",
+        "test_content",
+        "test_headers",
+        "test_error_message",
+        "uuid",
+        "url",
+        "created_at",
+        "modified_at",
+        "events_received",
+        "last_event_received_at",
```

Comment:
I dont think we should be collecting the following fields for analytics due to potential PII concerns-

1. name
2. additional_data_headers
3. test_content perhaps? Does this have potential to be sensitive/identifiable?
4. test_headers (if potential to be sensitive/identifiable)
5. url

Reply:
I think name and type will be what we will show?

The rest yeah, I agree these could be sensitive and we shouldn't collect it.

Reply:
@zkayyali812 I will collect event streams' names and types and skip others.

Reply:
Perfect, thank you @hsong-rh